admin/src/templates/edit_user.html

```diff
@@ -104,15 +104,20 @@
       <div class="help-block" translate>configuration.enable.token.login.refresh.help</div>
     </div>
 
-    <div class="form-group" ng-class="{ 'has-error': errors.password, 'required': !editUserModel.id, 'hidden': allowTokenLogin && (editUserModel.token_login || (editUserModel.token_login !== false && editUserModel.tokenLoginEnabled)) }">
+    <div class="form-group password-container" ng-class="{ 'has-error': errors.password, 'required': !editUserModel.id, 'hidden': allowTokenLogin && (editUserModel.token_login || (editUserModel.token_login !== false && editUserModel.tokenLoginEnabled)) }">
       <label for="edit-password" translate>Password</label>
-      <input id="edit-password" name="edit-password" type="password" class="form-control" ng-model="editUserModel.password" autocomplete="new-password" ng-disabled="editUserModel.role === '_admin'"/>
-      <span class="help-block">{{errors.password}}</span>
+      <div class="password-input-group">
+        <input id="edit-password" name="edit-password" type="{{editUserModel.passwordFieldType}}" class="form-control" ng-model="editUserModel.password" ng-disabled="editUserModel.role === '_admin'"/>
+        <button type="button" class="toggle-password" ng-click="togglePasswordMasking()">
+          <img ng-src="{{editUserModel.passwordFieldType === 'password' ? editUserModel.showPasswordIcon : editUserModel.hidePasswordIcon}}" alt="Toggle password masking" />
+        </button>
+        <span class="help-block">{{errors.password}}</span>
+      </div>
     </div>
 
     <div class="form-group" ng-class="{ 'required': !editUserModel.id, 'hidden': allowTokenLogin && (editUserModel.token_login || (editUserModel.token_login !== false && editUserModel.tokenLoginEnabled)) }">
       <label for="edit-password-confirm" translate>Confirm Password</label>
-      <input id="edit-password-confirm" name="edit-password-confirm" type="password" class="form-control" ng-model="editUserModel.passwordConfirm" autocomplete="new-password" ng-disabled="editUserModel.role === '_admin'"/>
+      <input id="edit-password-confirm" name="edit-password-confirm" type="{{editUserModel.passwordFieldType}}" class="form-control" ng-model="editUserModel.passwordConfirm"  ng-disabled="editUserModel.role === '_admin'"/>
```

Comment:
Is there a particular reason to remove the `autocomplete="new-password"` it's usually helpful to password manager plugins and the browser to know to suggest a new password.

Reply:
I removed the autocomplete="new-password" because I think I got some type of lint error when it was there along with the change to the 'type' attribute.  Without the 'type' attribute hard-coded to the value 'password', it couldn't tell that the type of the <input> element can be 'password' sometimes, so it marked the autocomplete="new-password" as incorrect.  Unfortunately, I forgot exactly where that error was.  I just tried adding the auto-complete attributes back in again.  Hopefully, that will either work or produce the error again.

Reply:
The error is now showing up in SonarCloud.